config/routes.rb

```diff
@@ -2,6 +2,7 @@
   resources :books, only: [:index]
   resources :authors, only: [:index]
 
+  root 'books#index'
```

Comment:
I have a rule, more specific routes at top, more global at bottom.
And also, you will look at root path maybe one time a year, I hope.